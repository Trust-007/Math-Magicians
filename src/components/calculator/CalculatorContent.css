.calc-content {
  display: flex;
  justify-content: center;
  align-items: flex-start;
  max-width: 100%;
  padding-inline: 10%;
  margin-top: 10%;
  padding-block: 3%;
}

.calc-content h1 {
  width: 100%;
  font-size: 30px;
  color: white;
}
