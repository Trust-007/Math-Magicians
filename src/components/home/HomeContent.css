.welcome {
  margin: 10% auto;
  width: 80%;
  background-color: rgb(24, 23, 23);
  margin-bottom: 2%;
}

.welcome h1 {
  font-size: 30px;
  color: aqua;
  margin-bottom: 3%;
}

.welcome p {
  color: white;
  font-size: 16px;
  margin-bottom: 50px;
}
