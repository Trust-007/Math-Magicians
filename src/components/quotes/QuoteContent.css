.quote {
  margin: auto;
  padding-top: 20%;
  width: 70%;
  font-size: 40px;
  color: white;
  height: 60vh;
}
