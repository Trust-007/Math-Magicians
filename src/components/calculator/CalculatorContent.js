import Calculator from './Calculator';
import './CalculatorContent.css';

const CalculatorContent = () => (
  <div className="calc-content">
    <h1>Lets do some math!</h1>
    <Calculator />
  </div>
);

export default CalculatorContent;
