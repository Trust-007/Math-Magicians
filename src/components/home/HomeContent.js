import './HomeContent.css';

const HomeContent = () => (
  <div className="welcome">
    <h1>Welcome to our page!!</h1>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias,
      aperiam sint cumque optio provident aliquid reiciendis quisquam
      perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur
      alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing
      elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita
      voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at
      non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing
      elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod
      accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa
      delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet
      consectetur adipisicing elit. Inventore illo earum, temporibus molestiae
      error, quibusdam praesentium vero sit facere veritatis ratione laudantium
      aliquam magni autem consectetur voluptates architecto suscipit facilis?
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo
      aspernatur ratione est eligendi impedit voluptas id, doloremque
      consectetur inventore omnis repudiandae debitis, dicta nisi tempora
      voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet
      consectetur adipisicing elit. Omnis dignissimos ratione quia
      necessitatibus perferendis minima vero doloremque odio dolorum obcaecati.
      Ipsum esse corporis nemo temporibus commodi necessitatibus qui est
      obcaecati.
    </p>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias,
      aperiam sint cumque optio provident aliquid reiciendis quisquam
      perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur
      alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing
      elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita
      voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at
      non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing
      elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod
      accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa
      delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet
      consectetur adipisicing elit. Inventore illo earum, temporibus molestiae
      error, quibusdam praesentium vero sit facere veritatis ratione laudantium
      aliquam magni autem consectetur voluptates architecto suscipit facilis?
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo
      aspernatur ratione est eligendi impedit voluptas id, doloremque
      consectetur inventore omnis repudiandae debitis, dicta nisi tempora
      voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet
      consectetur adipisicing elit. Omnis dignissimos ratione quia
      necessitatibus perferendis minima vero doloremque odio dolorum obcaecati.
      Ipsum esse corporis nemo temporibus commodi necessitatibus qui est
      obcaecati.
    </p>
  </div>
);

export default HomeContent;
