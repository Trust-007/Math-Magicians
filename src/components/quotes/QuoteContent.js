import './QuoteContent.css';

const QuoteContent = () => (
  <div className="quote">
    Mathematics is not about numbers, equations, computations or algorithms. Its
    about understanding. -William Paul Thurston.
  </div>
);

export default QuoteContent;
