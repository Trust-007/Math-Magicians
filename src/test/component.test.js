import React from 'react';
import renderer from 'react-test-renderer';

import CalculatorContent from '../components/calculator/CalculatorContent'
import HomeContent from '../components/home/HomeContent'
import QuoteContent from '../components/quotes/QuoteContent'

describe('testing components', () => {
    it('renders Calculator component correctly', () => {
        const tree = renderer.create(<CalculatorContent />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
    it('renders Home Component correctly', () => {
        const tree = renderer.create(<HomeContent />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
    it('renders Quote component correctly', () => {
        const tree = renderer.create(<QuoteContent />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})