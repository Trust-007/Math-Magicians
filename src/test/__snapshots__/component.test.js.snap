// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing components renders  correctly 1`] = `
<div
  className="calc-content"
>
  <h1>
    Lets do some math!
  </h1>
  <div
    className="container"
  >
    <button
      className="item1"
      onClick={[Function]}
      type="button"
    >
        
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="item0"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`testing components renders Calculator component correctly 1`] = `
<div
  className="calc-content"
>
  <h1>
    Lets do some math!
  </h1>
  <div
    className="container"
  >
    <button
      className="item1"
      onClick={[Function]}
      type="button"
    >
        
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="item0"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`testing components renders Home Component correctly 1`] = `
<div
  className="welcome"
>
  <h1>
    Welcome to our page!!
  </h1>
  <p>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias, aperiam sint cumque optio provident aliquid reiciendis quisquam perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Inventore illo earum, temporibus molestiae error, quibusdam praesentium vero sit facere veritatis ratione laudantium aliquam magni autem consectetur voluptates architecto suscipit facilis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo aspernatur ratione est eligendi impedit voluptas id, doloremque consectetur inventore omnis repudiandae debitis, dicta nisi tempora voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis dignissimos ratione quia necessitatibus perferendis minima vero doloremque odio dolorum obcaecati. Ipsum esse corporis nemo temporibus commodi necessitatibus qui est obcaecati.
  </p>
  <p>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias, aperiam sint cumque optio provident aliquid reiciendis quisquam perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Inventore illo earum, temporibus molestiae error, quibusdam praesentium vero sit facere veritatis ratione laudantium aliquam magni autem consectetur voluptates architecto suscipit facilis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo aspernatur ratione est eligendi impedit voluptas id, doloremque consectetur inventore omnis repudiandae debitis, dicta nisi tempora voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis dignissimos ratione quia necessitatibus perferendis minima vero doloremque odio dolorum obcaecati. Ipsum esse corporis nemo temporibus commodi necessitatibus qui est obcaecati.
  </p>
</div>
`;

exports[`testing components renders Quote component correctly 1`] = `
<div
  className="quote"
>
  Mathematics is not about numbers, equations, computations or algorithms. Its about understanding. -William Paul Thurston.
</div>
`;

exports[`testing components renders Quote component correctly 2`] = `
<div
  className="quote"
>
  Mathematics is not about numbers, equations, computations or algorithms. Its about understanding. -William Paul Thurston.
</div>
`;

exports[`testing components renders correctly 1`] = `
<div
  className="calc-content"
>
  <h1>
    Lets do some math!
  </h1>
  <div
    className="container"
  >
    <button
      className="item1"
      onClick={[Function]}
      type="button"
    >
        
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="item0"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="special"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`testing components renders correctly 2`] = `
<div
  className="welcome"
>
  <h1>
    Welcome to our page!!
  </h1>
  <p>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias, aperiam sint cumque optio provident aliquid reiciendis quisquam perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Inventore illo earum, temporibus molestiae error, quibusdam praesentium vero sit facere veritatis ratione laudantium aliquam magni autem consectetur voluptates architecto suscipit facilis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo aspernatur ratione est eligendi impedit voluptas id, doloremque consectetur inventore omnis repudiandae debitis, dicta nisi tempora voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis dignissimos ratione quia necessitatibus perferendis minima vero doloremque odio dolorum obcaecati. Ipsum esse corporis nemo temporibus commodi necessitatibus qui est obcaecati.
  </p>
  <p>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est alias, aperiam sint cumque optio provident aliquid reiciendis quisquam perferendis veritatis tenetur sed iusto hic minus. Impedit consectetur alias amet dolorem. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quis enim quibusdam, recusandae quisquam harum tempore, expedita voluptatibus sunt nulla corporis repellat magni. Explicabo animi esse at non nisi enim ratione. Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem reprehenderit laudantium voluptatibus veniam? Quod accusantium incidunt officia, praesentium nihil nisi ex dolor! Maxime ipsa delectus vero repellat, temporibus amet illum! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Inventore illo earum, temporibus molestiae error, quibusdam praesentium vero sit facere veritatis ratione laudantium aliquam magni autem consectetur voluptates architecto suscipit facilis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Non, quo aspernatur ratione est eligendi impedit voluptas id, doloremque consectetur inventore omnis repudiandae debitis, dicta nisi tempora voluptate! Laboriosam, amet laudantium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis dignissimos ratione quia necessitatibus perferendis minima vero doloremque odio dolorum obcaecati. Ipsum esse corporis nemo temporibus commodi necessitatibus qui est obcaecati.
  </p>
</div>
`;

exports[`testing components renders correctly 3`] = `
<div
  className="quote"
>
  Mathematics is not about numbers, equations, computations or algorithms. Its about understanding. -William Paul Thurston.
</div>
`;
